framework/src/base/ComputeUserObjectsThread.C

```diff
@@ -73,23 +73,21 @@ void
 ComputeUserObjectsThread::onElement(const Elem * elem)
 {
   _fe_problem.prepare(elem, _tid);
```

Comment:
Why do we need to retain the `prepare()` call here? Couldn't that be moved into the if clause?